.github/workflows/chart-lint-and-installation.yaml

```
@@ -15,14 +12,12 @@ on:
     branches-ignore:
       - 'dependabot/**'
   pull_request:
-    paths:
-      - "charts/**"
 
 permissions:
   contents: read
 
 jobs:
-  chart-lint-test:
+  chart-lint-and-installation-test:
```

```suggestion
  test-on-kubernetes:
    name: Test on Kubernetes
```
or 
```suggestion
  test-on-kubernetes-matrix:
    name: Test on Kubernetes
```
If there is a plan to run this test on multiple clusters.